scripts/prettier/prettier-helpers.js

```diff
@@ -14,12 +14,16 @@ const prettierExtensions = ['ts', 'tsx', 'js', 'jsx', 'json', 'scss', 'css', 'ht
 /**
  * Run prettier for a given set of files.
  *
- * @param {string[]} files List of files for which to run prettier
- * @param {boolean} [runAsync] Whether to run the command synchronously or asynchronously
- * @param {boolean} [logErrorsOnly] If true, log errors/warnings only. Otherwise log all output.
+ * @param {string[]} files - List of files for which to run prettier
+ * @param {Object} config
+ * @param {boolean=} config.runAsync - Whether to run the command synchronously or asynchronously
```

Comment:
What does the = mean here?

Reply:
JSDOC annotation for optional. Translated to ts => `{runAsync?: boolean}` 

In general I prefer to use TypeScript syntax for JSDoc, but when dealing with objects, the docs would not be properly parsed nor readable.

↓

```ts
/**
*
* @param {string[]} files - List of files for which to run prettier
* @param {{
*  /*  Whether to run the command  */ 
*   runAsync?: boolean
* }} config
*/
```